src/Servers/Kestrel/Core/src/Internal/Http3/Http3Stream.cs

```diff
@@ -593,7 +595,7 @@ private bool TryValidatePseudoHeaders()
             }
 
             // Approximate MaxRequestLineSize by totaling the required pseudo header field lengths.
-            var requestLineLength = _methodText.Length + Scheme.Length + hostText.Length + path.Length;
+            var requestLineLength = _methodText!.Length + Scheme!.Length + hostText.Length + path.Length;
```

Comment:
Nit:

```suggestion
            var requestLineLength = _methodText!.Length + _scheme.Length + hostText.Length + path.Length;
```

Really, all references to `Scheme` in this method should probably be to `_scheme`. This looks like a similar mistake to the one that was made in HttpProtocol.ProduceContinue where it references RequestHeaders instead of HttpRequestHeaders.

We should come up with a convention to make it clear with properties are user-settable and which ones are the internal representation.

Reply:
_scheme is private. If this will change then it should happen in another PR.